src/Assets.php

```diff
@@ -58,16 +58,18 @@ class_loaded: "lazyloaded",
             // Get the instance and puts it in the lazyLoadInstance variable
             var lazyLoadInstance = e.detail.instance;
         
-            var observer = new MutationObserver(function(mutations) {
-                mutations.forEach(function(mutation) {
-                    lazyLoadInstance.update();
+            if ("MutationObserver" in w) {
```

Comment:
`w` is not defined here, it should be `window`.
Personnaly I would write it like this: `if (window.MutationObserver) {`.